module/EntraBeta/AdditionalFunctions/New-EntraBetaPrivateAccessApplicationSegment.ps1

```diff
@@ -6,10 +6,10 @@ function New-EntraBetaPrivateAccessApplicationSegment {
 	[CmdletBinding()]
 	param (
 
-		[Alias('id')]
+		[Alias('ObjectId')]
 		[Parameter(Mandatory = $True, Position = 1, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
 		[string]
-		$ObjectID,
+		$ApplicationID,
```

Comment:
We need to be consistent with the parameter naming.
Either `ApplicationID` or `ApplicationId`

Reply:
Great catch.